### DISCUSSION TITLE
No Statistics Displaying in Dashboard

### DISCUSSION THREAD

**Discussion by ironorr44** [AdguardTeam/AdGuardHome#7153] [Q&A]
Hello!

I'm running version 0.107.52 in Debian 12. It seems to be working great (ad heavy pages are now nice and clean!) but there are no statistics showing in the dashboard. They appear to be enabled:

![Screenshot from 2024-07-25 22-56-53](https://github.com/user-attachments/assets/88b150ad-66d9-40cf-b4a3-9afade9b0e8a)

I enabled verbose logs and then visited a few ad heavy web pages to hopefully generate an error of some type. Does anyone have more experience reading the log file that might be able to see what I'm missing?

[aghlog.txt](https://github.com/user-attachments/files/16386036/aghlog.txt)

Any help is greatly appreciated!

**Comment by ainar-g** [comment]
What do you mean by “no statistics”?  Is everything just `0`, or are there some statsitics displayed but not others?

Are there records in the query log?

Also, does the browser console show any errors when you navigate to the dashboard or reload the page there?

**Reply by ironorr44** [reply]
The statistics panels all read 0, the query log is empty, and all the "Top ..." panels say "No ... data found". Screenshots attached.

I don't see any error messages when I refresh.

Thanks!

![Screenshot_20240726-075114.png](https://github.com/user-attachments/assets/cbe1f817-349a-4668-89dd-1ae931c7e45e)

![Screenshot_20240726-075207.png](https://github.com/user-attachments/assets/fba6638b-2c8a-48b4-b7be-0c9648361b75)

![Screenshot_20240726-075207.png](https://github.com/user-attachments/assets/1ac2bfc2-5b3b-4b8d-96f0-b79d4ef84129)

**Reply by ironorr44** [reply]
Is there a local database that stores the statistics data? I can check that to see if it's empty as a starting point.

**Reply by ironorr44** [reply]
I think I found the db in /opt/AdGuardHome/data/stats.db. Any tips on how I could check if it's collecting data?

On a whim, I tried chmod 777 to open up the permissions to test, but I'm still seeing all 0's in the web interface. Is it possible that the web interface isn't finding this db for some reason?

![Screenshot from 2024-08-03 21-48-20](https://github.com/user-attachments/assets/bad620d1-542e-4be0-831e-a6e83158797d)
